app/views/homepage/_government_activity.erb

```diff
@@ -0,0 +1,75 @@
+<!--
+Created a custom section and header pattern to
+- display a border-top of 2px (to replace .home-divider)
+- margin-bottom of 30px (both mobile and desktop)
+- padding-top to create a gap between border and heading
+-->
+<section class="homepage-section homepage-section__government-activity">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds" data-module="gem-track-click">
+      <div class="homepage-section__heading">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t("homepage.index.government_activity"),
+          margin_bottom: 2,
+        } %>
+
+        <p class="govuk-body-l">
+          <%= t("homepage.index.government_activity_description") %>
+        </p>
+      </div>
+
+      <ul class="homepage-services-and-info__list">
+        <% t("homepage.government_activity").each do | item | %>
+          <%= render partial: "homepage/chevron_card", locals: {
+            description: item[:description],
+            link: item[:link],
+            title: item[:title],
+            track_action: "governmentactivityLink",
+          } %>
+        <% end %>
+      </ul>
+    </div>
+
+    <div class="govuk-grid-column-one-third">
+      <div class="homepage-section__heading">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t("homepage.index.departments_and_organisations"),
+        } %>
+      </div>
+      
+      <div class="homepage-section__depts" data-module="gem-track-click">
+        <div class="homepage-big-number__wrapper">
+          <%= render "govuk_publishing_components/components/big_number", {
+            number: t("homepage.index.ministerial_departments_count"),
+            label: t("homepage.index.ministerial_departments"),
+            href: "/government/organisations#ministerial_departments",
+            data_attributes: {
+              "track-category": "homepageClicked",
+              "track-action": "departmentsLink",
+              "track-label": "/government/organisations#ministerial_departments",
+              "track-dimension": "#{t("homepage.index.ministerial_departments_count")} #{t("homepage.index.ministerial_departments")}",
+              "track-dimension-index": 29,
+              "track-value": 1,
+            },
+          } %>
+        </div>
+
+        <div class="homepage-big-number__wrapper">
+          <%= render "govuk_publishing_components/components/big_number", {
+            number: t("homepage.index.other_agencies_count"),
+            label: raw(t("homepage.index.other_agencies")),
```

Comment:
A teeny weeny nitpick that this was originally `raw` because of the need for the text to go over 2 lines so it looks nice. I'm pretty confident that we don't need it anymore.

Reply:
Yep that's good. Makes no difference when removed (I've also removed an `&nbsp;` from the locale entry too)